docs/intro/quickstart/modeling.rst

```diff
@@ -0,0 +1,51 @@
+.. _ref_quickstart_modeling:
+
+=================
+Modeling our data
+=================
+
+.. edb:split-section::
+
+  Our flashcards application has a simple data model, but it's interesting enough to get a taste of many of the features of the Gel schema language. We have a ``Card`` type that describes an single flashcard, which for now contains two required string properties: ``front`` and ``back``. Each ``Card`` belongs to a ``Deck``, and there is an explicit ordering to the cards in a given deck.
+
+  Starting with this simple model, let's express these types in the ``default.gel`` schema file.
+
+  .. code-block:: sdl-diff
+    :caption: dbschema/default.gel
+
+      module default {
+    +   type Deck {
+    +     required name: str;
+    +     description: str;
+    +   };
+
+    +   type Card {
+    +     required order: int64;
```

Comment:
I wonder if we should explain `order` a bit more - can be me, but I only got it after reading the next chapters 😅

Reply:
Yeah, good call, we can say that link sets are unordered, and since we want the cards to have a fixed order, we add this property so we can order them when we query for the deck and its cards.